app/src/main/java/gov/hhs/cdc/trustedintermediary/external/slf4j/DeployedLogger.java

```diff
@@ -0,0 +1,83 @@
+package gov.hhs.cdc.trustedintermediary.external.slf4j;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.util.Arrays;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+/**
+ * Humble object interface for logging. Uses SLF4J behind the scenes. The deployed logger doesn't
+ * colorize its messages and uses a logger name that uses the JSON structured logging.
+ */
+public class DeployedLogger implements Logger {
+
+    private static final DeployedLogger INSTANCE = new DeployedLogger();
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger("tilogger");
+
+    private DeployedLogger() {}
+
+    public static DeployedLogger getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    public void logTrace(String traceMessage) {
+        Level level = Level.TRACE;
+        LoggerHelper.logMessageAtLevel(LOGGER, level, traceMessage).log();
+    }
+
+    @Override
+    public void logDebug(String debugMessage) {
+        Level level = Level.DEBUG;
+        LoggerHelper.logMessageAtLevel(LOGGER, level, debugMessage).log();
+    }
+
+    @Override
+    public void logDebug(String debugMessage, Throwable e) {
```

Comment:
Are there use cases for logging an exception at debug level? I can only think of uses at error/fatal level

Reply:
Yeah, this method is not even used.  I think I'll remove all the unused methods.  If we need them later, we can add them then.  Thoughts?

Reply:
That sounds good to me. Same for the one in LocalLogger.java if it's not used. If there's an exception, I think either error or fatal level is more appropriate, unless there is some use case I'm not taking into consideration